codealike.json

```diff
@@ -0,0 +1 @@
+{"projectId":"6944aade-3195-477c-8aed-9ebe2902fc57","projectName":"RavenDB","neverTrack":false}
```

Comment:
Probably this should be removed, no?

Reply:
I use Codealike to measure myself, that is the correlation token (it used to exist at the solution level)  after many requests we moved it to a standalone file. :)